packages/driver/src/cy/multidomain/index.ts

```diff
@@ -26,6 +26,27 @@ export function addCommands (Commands, Cypress: Cypress.Cypress, cy: Cypress.cy,
     // this isn't fully implemented, but in place to be able to test out
```

Comment:
I'm assuming this comment is still accurate?

Reply:
Yes, I'm working on it for #18275.